Content.Server/Sound/EmitSoundSystem.cs

```diff
@@ -1,3 +1,4 @@
+#nullable enable
```

Comment:
#nullable enable not needed in Content.Server, nullables are enabled project-wide